### ISSUE TITLE
Missing Pydantic in requirements python sdk

### ISSUE THREAD

**Issue by allan-gon** [mendableai/firecrawl#1012]
**Describe the Bug**
Missing Pydantic in requirements python sdk

**To Reproduce**
Steps to reproduce the issue:
1. Run pip install firecrawl-py
2. Instantiate app
3. Run the script and experience the error saying pydantic not installed

**Expected Behavior**
Should just work. That is not error out because its missing a requirment when installing

**Screenshots**
![image](https://github.com/user-attachments/assets/e71cfea2-2944-45f9-8e28-f6e0565c1bb2)

**Environment (please complete the following information):**
- Windows 10
- Firecrawl Version: 1.6.8

**Logs**
output:
```
Traceback (most recent call last):
  File "C:\Users\Allan\workspace\dopai\client_app\main.py", line 1, in <module>
    from firecrawl import FirecrawlApp
  File "C:\Users\Allan\.virtualenvs\client_app-E4E_U366\lib\site-packages\firecrawl\__init__.py", line 14, in <module>
    from .firecrawl import FirecrawlApp # noqa
  File "C:\Users\Allan\.virtualenvs\client_app-E4E_U366\lib\site-packages\firecrawl\firecrawl.py", line 19, in <module>
    import pydantic
ModuleNotFoundError: No module named 'pydantic'
```

**Solution**
Add pydantic to requirements for python sdk

**Comment by RutamBhagat** [comment]
#990 addresses this pydantic requirement

**Comment by nickscamara** [comment]
Merged! Live on 1.7.0! Thanks @allan-gon and @RutamBhagat for the pr.
